trackma/ui/gtk/window.py

```diff
@@ -357,8 +357,9 @@ def _on_about(self, action, param):
         about.set_copyright("© z411, et al.")
         about.set_authors(["See AUTHORS file"])
         about.set_artists(["shuuichi"])
-        about.connect('destroy', self._on_modal_destroy)
-        about.present()
+        about.connect('destroy-event', self._on_modal_destroy)
+        about.connect('response',lambda x,y: about.destroy())
+        about.run()
```

Comment:
This should be in another pull request.

Also, it should append before run, so it'll be something like this instead:

```diff
         about.set_authors(["See AUTHORS file"])
         about.set_artists(["shuuichi"])
         about.connect('destroy', self._on_modal_destroy)
-        about.present()
+        about.connect('response', lambda dialog, response: dialog.destroy())
         self._modals.append(about)
+        about.run()

     def _on_modal_destroy(self, modal_window):
         self._modals.remove(modal_window)
```